ui/src/pipeline/PipelineEdit.vue

```diff
@@ -31,6 +31,11 @@
                 label="Pipeline Name"
                 :rules="[required]"
               />
+              <v-text-field
+                v-model="dialogPipeline.datasourceId"
+                label="Referenced Datasource Id"
+                :rules="[required]"
+              />
             </v-form>
             <pipeline-edit-stepper-button-group
```

Comment:
Maybe we can use a single page edit instead of stepper now that the pipeline edit is so much shorter. But this is for another pull request..